pyiron_atomistics/sphinx/base.py

```diff
@@ -388,16 +388,16 @@ def load_default_input(self):
         sph.create_group("initialGuess")
         sph.create_group("main")
 
-        self.input.EnCut = 340
+        self.input.EnCut = 20 * RYDBERG_TO_EV
```

Comment:
These changes are not related to the idea of this PR, correct? I'd request them to be done in a different PR. Btw. these changes probably make the tests fail.